iot_key_employee_rfid/models/hr_employee.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2019 Creu Blanca
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import _, api, fields, models
+
+
+class HrEmployee(models.Model):
+    _name = "hr.employee"
+    _inherit = ["hr.employee", "iot.key.mixin"]
+
+    iot_key_id = fields.Many2one("iot.key", compute="_compute_iot_key")
+    iot_key_ids = fields.One2many(context={"active_test": False}, string="IoT Keys")
+    rule_ids = fields.Many2many(
+        "iot.rule", related="iot_key_id.rule_ids", readonly=False, string="IoT Rules",
+    )
+    rfid_card_code = fields.Char(
+        store=True,
+        compute="_compute_rfid_card_code",
+        inverse="_inverse_rfid_card_code",
+    )
+
+    @api.depends("iot_key_ids.unique_virtual_key", "iot_key_ids")
+    def _compute_rfid_card_code(self):
+        for record in self:
+            if record.iot_key_ids:
+                record.rfid_card_code = record.iot_key_ids.unique_virtual_key
+
```

Comment:
a compute must return always something
```suggestion
else:
    record.rfid_card_code=False

```

Reply:
You don't need the if, if iot_key_ids is not null it will return the expected value. However, if two keys exists, a problem might arise... So, I would propose to do 
`record.rfid_card_code = record.iot_key_ids[:1].unique_virtual_key`